ui/raidboss/data/05-shb/trial/ruby_weapon-ex.js

```diff
@@ -269,7 +269,7 @@ export default {
       netRegexCn: NetRegexes.ability({ source: '红宝石神兵', id: '4AFC', capture: false }),
       netRegexKo: NetRegexes.ability({ source: '루비 웨폰', id: '4AFC', capture: false }),
       preRun: function(data) {
-        for (color of data.colors) {
+        for (let color of data.colors) {
```

Comment:
I know this isn't your code, but I think this is intending to switch all the values of `data.colors`, e.g. something more like `for (const id of data.colors) { data.colors[id] = data.colors[id] === 'red' ? 'blue' : 'red'; }`.